charter.md

```diff
@@ -0,0 +1,45 @@
+There are many situations in which it is desirable to take
+measurements of data which people consider sensitive. For instance,
+one might want to measure the most common sites that people browse to
+or exposure to some disease. In these cases, the entity taking the
+measurement is not interested in people's individual responses but
+rather in aggregated data (e.g., how many users visit URL X), but
+conventional methods require collecting individual responses and then
+aggregating them, thus representing a threat to user privacy and
+rendering many such measurements difficult and impractical.
+
+New cryptographic techniques such as Prio and Heavy Hitters address
```

Comment:
"Heavy Hitters" isn't the name of the protocol in ePrint 2021/017. "Prio" is the name of a protocol, but we don't implement it. (We actually implement ePrint 2019/088.) Rather than name the protocols, I suggest we just cite the papers.

Reply:
I think perhaps we just say "Prio". We're not saying we're going to standardize that, we're giving an example.

Reply:
Also, I'm trying to get HCG to just rename all this stuff Prio